docs/platform/concepts/enhanced-compliance-env.md

```diff
@@ -23,9 +23,8 @@ following are requirements to utilize an ECE:
 
 -   A BAA signed with Aiven
 -   Plan to operate in one or more of Amazon Web Services (AWS), Google
-    Cloud Platform (GCP), or Microsoft Azure
+    Cloud, or Microsoft Azure
 -   Total monthly spend is greater than \$5,000
--   An active Enterprise Support Contract
```

Comment:
There is a support requirement for a customer wanting ECE, but the specific tier needs to be confirmed

Reply:
Support is no longer an ECE requirement, and can be removed